src/confd/share/migrate/1.3/10-ssh-server.sh

```diff
@@ -3,14 +3,17 @@
 
 file=$1
 temp=${file}.tmp
-
-jq '.["infix-services:ssh"] = {
-      "enabled": true,
-      "hostkey": ["genkey"],
-      "listen": [
-        {"name": "ipv4", "address": "0.0.0.0", "port": 22},
-        {"name": "ipv6", "address": "::", "port": 22}
-      ]
-    }' "$file" > "$temp"
-
-mv "$temp" "$file"
+jq '
+if has("infix-services:ssh") then
+  .
+else
+  .["infix-services:ssh"] = {
```

Comment:
Sorry for being such a bother, but what happens when a user does:

   > configure
   > no ssh
   > leave

Followed by a reboot? Doesn't this re-enable ssh on port 22 then?

Reply:
> Sorry for being such a bother, but what happens when a user does:
> 
> 
> 
>    > configure
> 
>    > no ssh
> 
>    > leave
> 
> 
> 
> Followed by a reboot? Doesn't this re-enable ssh on port 22 then?

Not if the version is stepped up, this, this is what we discussed earlier.

Reply:
> Sorry for being such a bother, but what happens when a user does:
> 
> 
> 
>    > configure
> 
>    > no ssh
> 
>    > leave
> 
> 
> 
> Followed by a reboot? Doesn't this re-enable ssh on port 22 then?

Not if the version is stepped up, this, this is what we discussed earlier.

Reply:
Ah, of course. Thanks!